03-Custom-Actions-In-Python/02-An-Initial-Custom-Action.ipynb

```diff
@@ -0,0 +1,249 @@
+{
```

Comment:
<p>Suggest moving this to boilerplate code (as in the other notebooks) to avoid interrupting the flow in the description of operations.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/glotzerlab/hoomd-examples/pull/17/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>